src/status/contacts.nim

```diff
@@ -9,6 +9,8 @@ const DELETE_CONTACT* = "__deleteThisContact__"
 
 type
   ContactModel* = ref object
+    # contact.ID => Profile
+    contactsMap: Table[string, Profile]
```

Comment:
Is there a chance that the contactsMap could be accessed asynchronously? If so, I wonder if it could cause crashes.

Reply:
That's why I need your review, since you know the codebase much better than me.  Who could confirm this code can be accessed asynchronously or not?